packages/web-components/src/components/content-item-horizontal/__stories__/content-item-horizontal.stories.ts

```diff
@@ -63,6 +64,34 @@ export const Default = ({ parameters }) => {
   `;
 };
 
+export const WithThumbnail = ({ parameters }) => {
+  const { alt, heading, copy, ctaType1, ctaCopy1, href1, ctaType2, ctaCopy2, href2 } =
+    parameters?.props?.ContentItemHorizontal ?? {};
+  return html`
+    <dds-content-item-horizontal thumbnail="true">
+      <dds-content-item-heading>${heading}</dds-content-item-heading>
+      <dds-content-item-horizontal-copy>${copy}</dds-content-item-horizontal-copy>
+      <dds-link-list slot="footer" type="vertical">
+        <dds-link-list-item-cta
+          icon-placement="${ICON_PLACEMENT.RIGHT}"
+          href="${ifNonNull(href1)}"
+          cta-type="${ifNonNull(ctaType1)}"
+        >
+          ${ctaCopy1}
+        </dds-link-list-item-cta>
+        <dds-link-list-item-cta
+          icon-placement="${ICON_PLACEMENT.RIGHT}"
+          href="${ifNonNull(href2)}"
+          cta-type="${ifNonNull(ctaType2)}"
+        >
+          ${ctaCopy2}
+        </dds-link-list-item-cta>
+      </dds-link-list>
+      <dds-image slot="thumbnail" alt="${ifNonNull(alt)}" default-src="${imgMd4x3}"></dds-image>
```

Comment:
@shixiedesign or @pavolprok Do we need to account for media queries for this thumbnail image? I'm not sure if there are different ratios or dimensions that we need to serve on different breakpoints.

Reply:
Hi Putra. There will be only 4:3 ratio for the thumbnail images. The thumbnail images should span across 3 last columns  on max, xlg, lg, md. The only chage it rems of placement of the images is on the small breakpoint where the images spans across 3 columns but is aligned to the left and is placed below the CTA.